geode-core/src/main/java/org/apache/geode/internal/cache/BucketRegion.java

```diff
@@ -557,6 +557,34 @@ public boolean virtualPut(EntryEventImpl event, boolean ifNew, boolean ifOld,
     }
   }
 
+  @Override
+  void cmnClearRegion(RegionEventImpl regionEvent, boolean cacheWrite, boolean useRVV) {
+    if (getBucketAdvisor().isPrimary()) {
+      logger.info("Not primary bucket when doing clear, do nothing");
+      return;
+    }
+
+    boolean enableRVV = useRVV && getConcurrencyChecksEnabled();
+    RegionVersionVector rvv = null;
+    if (enableRVV) {
+      rvv = getVersionVector();
```

Comment:
Does this call just return null if it's not enabled?  Do we need the if check?

Reply:
Yes, then just use null for rvv.

Reply:
If this returns null, then the if check isn't needed.  It'll be null either way.

Reply:
There's basicLocalClear whose useRVV is false. I need to consider that.

Reply:
Lets think of the case where useRVV is false.
if (false) {} 
rvv is already null, so rvv ends up null;
OR
//remove the if check
rvv = getVersionVector() <-returns null anyways

The end result is the same.  why have the if check?

Reply:
In BR.cmnClearRegion(), since we have locked the rvvLock, no more operations will be intiated here. So no need to explicitly use RVV. I will fix this part.